generators/server/templates/src/test/java/package/web/rest/_LogsResourceIntTest.java

```diff
@@ -19,10 +19,11 @@
 package <%= packageName %>.web.rest;
 
 <%_ if (databaseType === 'cassandra') { _%>
-import <%= packageName %>.AbstractCassandraTest;
+import <%=packageName%>.AbstractCassandraTest;
 <%_ } _%>
-import <%= packageName %>.<%= mainClass %>;
-import <%= packageName %>.web.rest.vm.LoggerVM;
+import <%=packageName%>.<%= mainClass %>;<% if (authenticationType == 'uaa') { %>
```

Comment:
What worries me the most are those ugly red squares in github...
Anyway, I don't really care so I'll do the space change.
Should we have a vote for the spacing policy :trollface: ?